packages/trader/src/Modules/CFD/Components/cfd-account-card.tsx

```diff
@@ -21,24 +29,29 @@ const account_icons = {
     },
 };
 
-const AddTradeServerButton = React.forwardRef(({ onSelectAccount, is_disabled }, ref) => {
-    return (
-        <div
-            onClick={is_disabled ? null : onSelectAccount}
-            className={classNames('cfd-account-card__add-server', {
-                'cfd-account-card__add-server--disabled': is_disabled,
-            })}
-            ref={ref}
-        >
-            <span className='cfd-account-card__add-server--icon'>+</span>
-            <Localize i18n_default_text='Add region' />
-        </div>
-    );
-});
+const AddTradeServerButton = React.forwardRef(
+    (
+        { onSelectAccount, is_disabled }: { onSelectAccount: () => void; is_disabled: boolean },
+        ref: React.LegacyRef<HTMLDivElement> | undefined
+    ) => {
+        return (
+            <div
+                onClick={is_disabled ? () => null : onSelectAccount}
```

Comment:
one more suggestion ☺️:
```suggestion
                onClick={is_disabled ? () => undefined : onSelectAccount}
```

Reply:
Undefined typically refers to something which has not yet been assigned a value (yet). Null refers to something which definitively has no value. In this case, I suppose returning a null makes more sense.

Reply:
@mahdiyeh-fs According to the Typescript docs for react v16 (we are currently using 16.14.0), the onClick() property of a JSX.Element can be assigned with either a function or undefined.
[        onClick?: MouseEventHandler<T> | undefined;](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fe2e2b7f8535855125db808c55abe4a3bdf13a28/types/react/v16/index.d.ts#L1455)